#!/bin/sh /usr/share/dpatch/dpatch-run
## 324-magic-add-snes.dpatch by Josh Triplett <josh@freedesktop.org>
##
## DP: New magic for SNES SPC700 sound files (Closes: #410847).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/audio file-4.21/magic/Magdir/audio
--- file-4.21.orig/magic/Magdir/audio	2007-12-23 17:32:42.000000000 +0000
+++ file-4.21/magic/Magdir/audio	2007-12-23 17:33:57.000000000 +0000
@@ -268,6 +268,15 @@
 >122	byte&0x1	=1		PAL
 >122	byte&0x1	=0		NTSC
 
+# Type: SNES SPC700 sound files
+# From: Josh Triplett <josh@freedesktop.org>
+0	string	SNES-SPC700\ Sound\ File\ Data\ v	SNES SPC700 sound file
+>&0	string	0.30					\b, version %s
+>>0x23	byte	0x1B					\b, without ID666 tag
+>>0x23	byte	0x1A					\b, with ID666 tag
+>>>0x2E	string	>\0					\b, song "%.32s"
+>>>0x4E	string	>\0					\b, game "%.32s"
+
 # Impulse tracker module (audio/x-it)
 0	string		IMPM		Impulse Tracker module sound data -
 >4	string		>\0		"%s"
